tests/unit/api/test_api.py

```diff
@@ -141,7 +141,7 @@ def test_hostname_warning_not_present(self):
     def test_normalize_tags(self):
         tag_list_test = ["tag1, tag2", "tag3 ,tag4", "tag5,tag6"]
         tag_list_final = normalize_tags(tag_list_test)
-        assert tag_list_final == ["tag1_ tag2", "tag3 _tag4", "tag5_tag6"]
+        assert tag_list_final == ["tag1__tag2", "tag3__tag4", "tag5_tag6"]
```

Comment:
in case we want to keep space https://github.com/DataDog/datadogpy/pull/517/files#r385638649
```suggestion
        assert tag_list_final == ["tag1_ tag2", "tag3 _tag4", "tag5_tag6"]
```